trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -147,7 +149,7 @@ describe('controllers - case', () => {
 
       describe('when deal cancellation is disabled', () => {
         it('should render the template with showDealCancelButton=false', async () => {
-          jest.mocked(isDealCancellationEnabled).mockReturnValueOnce(false);
+          jest.mocked(canDealBeCancelled).mockReturnValueOnce(false);
```

Comment:
Can we update the test descriptions please @BethThomas141 , they're currently:

- when deal cancellation is enabled
- when deal cancellation is disabled

(etc)

But the condition is now "is deal cancellation permitted"

Reply:
Great spot, have changed! Thanks!